Makefile

```
@@ -58,3 +58,34 @@ verify-licenses: | $(GOLICENSES)
       rm -rf temp; \
       exit 1; \
     fi; \
+
+.PHONY: fixture-gorillamux
+fixture-gorillamux:
+	LIBRARY=gorillamux $(MAKE) fixture
+
+.PHONY: fixture-nethttp
+fixture-nethttp:
+	LIBRARY=nethttp $(MAKE) fixture
+
+.PHONY: fixture
+fixture:
```

This will add a `fixture` target that shouldn't be called (`LIBRARY` will not be set if it is directly called). Instead, we could compose the built target

```suggestion
.PHONY: fixture-nethttp fixture-gorillamux
fixture-nethttp: fixtures/nethttp
fixture-gorillamux: fixtures/gorillamux
fixture/%: LIBRARY=$*
fixture/%:
```